go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/sirupsen/logrus v1.3.0
 	github.com/weppos/publicsuffix-go v0.4.0
 	github.com/zmap/zcrypto v0.0.0-20190329181646-dff83107394d
+	golang.org/x/crypto v0.0.0-20180904163835-0709b304e793
```

Comment:
@tadukurow This looks like an older commit of `golang.org/x/crypto` than was in the `go.mod` before this PR. Is there a reason you had to revert to an older version or could this diff be backed out of this branch?

Reply:
D'oh, I was comparing this to the `zcrypto` ref. I see now this isn't a change in an import version but a new addition. Apologies.